src/components/Keyword/AddKeyword/index.js

```diff
@@ -170,56 +194,69 @@ const AddKeyword = () => {
   }, [keywordRecommendationList]);
 
   return (
-    <AddKeywordContainer>
+    <AddKeywordContainer toggle={isOpen}>
       <KeywordHeader title={'키워드 추가하기'} />
       <AddKeywordContent>
         <S.HashtagContainer toggle={isOpen} keyword={recommendKeyword === ''} alreadyRegister={isRegisterKeyword}>
           <S.HashtageImage src="/asset/hashtagblack.svg" alt="hashtage_image" />
```

Comment:
```suggestion
          <S.HashtageImage src="/asset/Hashtagblack.svg" alt="hashtag_image" />
```

이미지 주소가 안맞고, alt 이름도 수정이 필요해 보입니다.

Reply:
여기 뿐 아니라 프로젝트 내에서 검색하면 '**HashTage**'라는 오타가 많이 나오는데 수정이 필요해 보이네요.

Reply:
전체적으로 이미지 파일에 대한 네이밍은 수정을 진행하겠습니다